### PR TITLE
Use ArgumentNullException.ThrowIfNull in PresentationUI and Themes

### PR DESCRIPTION
**Pull request by halgab**
## Description

Use ArgumentNullException.ThrowIfNull in `PresentationUI` and `Themes` as in #7181

## Regression

No

## Testing

CI

## Risk

Low: changes are mostly mechanic, using fixer CA1510

Note that many files don't follow indentation conventions, so I had to manually realign what the analyzer changed.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/wpf/pull/8194)

### CODE REVIEW

File: src/Microsoft.DotNet.Wpf/src/PresentationUI/MS/Internal/Documents/Application/TransactionalPackage.cs [hunk 1]
```diff
@@ -64,12 +64,9 @@ internal class TransactionalPackage : Package, IDisposable
...
         : base(FileAccess.ReadWrite)
     {
-        if (original == null)
-        {
-            throw new ArgumentNullException("original");
-        }
+            ArgumentNullException.ThrowIfNull(original);
 
-        Package originalPackage = Package.Open(original);
+            Package originalPackage = Package.Open(original);
```
Comment (miloush) [inline src/Microsoft.DotNet.Wpf/src/PresentationUI/MS/Internal/Documents/Application/TransactionalPackage.cs:69]: whitespace (and line 67)

File: src/Microsoft.DotNet.Wpf/src/PresentationUI/MS/Internal/Documents/RightsManagementProvider.cs [hunk 2]
```diff
@@ -448,14 +448,11 @@ RightsManagementUser IRightsManagementProvider.GetDefaultCredentials()
...
-        if (user == null)
-        {
-            throw new ArgumentNullException("user");
-        }
+        ArgumentNullException.ThrowIfNull(user);
 
         //Get AvailableCreds list so we can match new default user.
         ReadOnlyCollection<RightsManagementUser> users =
-            ((IRightsManagementProvider)this).GetAvailableCredentials();
+        ((IRightsManagementProvider)this).GetAvailableCredentials();
```
Comment (miloush) [inline src/Microsoft.DotNet.Wpf/src/PresentationUI/MS/Internal/Documents/RightsManagementProvider.cs:455]: whitespace

**Comment by miloush** [comment]
LGTM

**Comment by rchauhan18** [comment]
Thanks @halgab for your contribution.

**Pull request merged**